fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java

```diff
@@ -108,13 +98,12 @@ public static ProfileManager getInstance() {
     }
 
     private ProfileManager() {
-        lock = new ReentrantReadWriteLock(true);
+        // Only protect profileDeque; profileMap is concurrent, no need to protect
```

Comment:
It seems that this comment could be removed, since `profileDeque` is removed and `profileMap` isn't concurrent anymore.